src/common.ts

```diff
@@ -2,52 +2,132 @@ export let oDataQueryNames = ["select", "expand", "orderby", "filter", "top", "s
 export const DEFAULT_VERSION = "v1.0";
 export const GRAPH_BASE_URL = "https://graph.microsoft.com/";
 
-// NOTE: This should be kept up to date with the version used in package.json.
-// If you are changing this please ensure you are also changing it in package.json.
+/**
+ * @constant
+ * A package version
+ * @NOTE: This should be kept up to date with the version used in package.json.
+ * If you are changing this please ensure you are also changing it in package.json.
+ */
 export const PACKAGE_VERSION = "1.2.0";
 
-// support oData params with and without $ prefix
-oDataQueryNames = oDataQueryNames.concat(oDataQueryNames.map((s) => "$"+s));
+/**
+ * @extends
+ * support oData params with and without $ prefix
+ */
+oDataQueryNames = oDataQueryNames.concat(oDataQueryNames.map((s) => "$" + s));
```

Comment:
Not part of this PR but we should include search and format as values in oDataQueryNames

Reply:
Yeah that is something we have in our plate. will make a different PR for that.